process/outportBlockConverter.go

```diff
@@ -43,6 +44,45 @@ func NewOutportBlockConverter(
 	}, nil
 }
 
+// HandleMetaOutportBlock will convert an outport.OutportBlock to data.MetaOutportBlock.
+func (o *outportBlockConverter) HandleMetaOutportBlock(outportBlock *outport.OutportBlock) (*hyperOutportBlocks.MetaOutportBlock, error) {
+	headerType := outportBlock.BlockData.HeaderType
+
+	// check if the header type is supported by this function.
+	if headerType != string(core.MetaHeader) {
+		return nil, fmt.Errorf("cannot convert to meta outport block. header type: %s not supported", outportBlock.BlockData.HeaderType)
```

Comment:
```suggestion
		return nil, fmt.Errorf("cannot convert to meta outport block. header type: %s not supported", headerType)
```
use `headerType` here?